src/Microsoft.MonoDevelop.LanguageServices.Razor/Microsoft.MonoDevelop.LanguageServices.Razor.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Nothing in this project today. It's just a placeholder for VS for mac specific code for when it arises.